src/gui/FxMixerView.cpp

```diff
@@ -395,6 +395,40 @@ void FxMixerView::deleteChannel(int index)
 
 
 
+void FxMixerView::deleteUnusedChannels()
+{
+	TrackContainer::TrackList tracks;
+	tracks += Engine::getSong()->tracks();
+	tracks += Engine::getBBTrackContainer()->tracks();
+
+	// go through all FX Channels
+	for(int i = m_fxChannelViews.size()-1; i>0; --i)
```

Comment:
Ok, adjusted. But for some reason this isn't consistently used everywhere in the code. Sometimes there are spaces and sometimes not.